extensions/wikia/ImageReview/AbandonedWorkResetter.class.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+class AbandonedWorkResetter extends WikiaModel {
```

Comment:
A big part of the PR was breaking apart a couple giant classes into smaller, more clearly defined classes. This `AbandonedWorkResetter` is one of those more single purpose classes. All of it's code was pulled directly from `ImageReviewHelper`